src/streamlink/stream/hls_playlist.py

```diff
@@ -102,22 +103,22 @@ def parse_attributes(self, value):
         def map_attribute(key, value, quoted):
             return (key, quoted or value)
 
-        attr = re.findall(ATTRIBUTE_REGEX, value)
+        attr = M3U8Parser._attr_re.findall(value)
```

Comment:
Can you make these `self.` instead of `M3U8Parser.` - for subclassing, as we discussed in the previous PR?

Reply:
Done.